Biahun_Uladzimir/05.Ansible.run/keys/id_rsa_ansible.pub

```diff
@@ -0,0 +1 @@
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCw0+uRi8XNJwwa6Pq0/ALUaIFMVo4/qIs8IAuPbs8+2ZoXNK8vFS7Yfib6zlH6KZq96owZfIBaBU9Jt3V+hc8RSgRDQNnOX+C+ciMH8hALGNtteQR7TZxxZMGM9PtgYE/Wmn4ukenz+jAepq+AIP/rjNqsu9kdiL3mgrk1fBThqiVetCUKDQGKFVOA/oBhXfU+jUAnuSfCTcz/RgAGDSrhRVBJEJ6KIv6LhX6vqJ2l9ET5D5ykclgjvs0HdQG9VCAmeghgipJ3v0f+0DxENCLEX8Uwj2UFba1rZzTN+dQMuQLy6mNPq9LFMIhZSJGgybmFB89bGQSymx8iO8tVKjrj ansible_user@user
```

Comment:
Why is it so open? 
Could you please use vault

Reply:
I have move key to my homedir )